DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/DateBoxBase/1 Configuration/useMaskBehavior.md

---
---
##### shortDescription

> Specifies whether to limit user input with a mask created based on the [displayFormat](/api-reference/10%20UI%20Components/DateBoxBase/1%20Configuration/displayFormat.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#displayFormat') property. 

Reviewer: ```suggestion
Specifies whether to use an input mask based on the [displayFormat](/api-reference/10%20UI%20Components/DateBoxBase/1%20Configuration/displayFormat.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#displayFormat') property. 
```
